experimental/dds/tree2/src/index.ts

```diff
@@ -247,6 +248,9 @@ export {
 	NodeExistsConstraint,
 	cursorForTypedTreeData,
 	NodeExistenceStateChange,
+	defaultGetFieldGenerator,
```

Comment:
Do we want these to be exposed publicly? Do we expect end-users to implement field generators?